akka-docs/src/main/paradox/scala/remoting.md

```diff
@@ -594,7 +594,8 @@ While Akka is a peer-to-peer technology, each connection between nodes starts ou
 the other (the "server").
 
 Note that if TLS is enabled with mutual authentication there is still a risk that an attacker can gain access to a valid certificate
-by compromising any node with certificates issued by the same internal PKI tree.
+by compromising any node with certificates issued by the same internal PKI tree. To limit this risk hostname validation can be enabled.
```

Comment:
I don't think that hostname verification is improving the situation as much as this edit suggests.   Hostname verification improves the trustworthiness of the passive side but not the active side of the peer-to-peer relationship.    It will still be the case that any client with a certificate issued by the same PKI can connect, since Akka has no authorization logic (to my knowledge) beyond 'is authenticated'.

In other words, the root cause of the issue alluded to by the above security note is a lack of authorization logic in Akka.  Could the note be clarified to that effect?